service/modules/synthetic-data-module/README.md

```diff
@@ -0,0 +1,22 @@
+# Produce synthetic data from a document, evaluate with sdgym
+
+## Installing sdgym
+
+### Pip install
+
+Note: Make sure you have Cython installed, as well as scikit learn.

```

Comment:
We should have a Dockerfile for this soon. I may ping you on what would be needed for it